force-app/main/default/classes/FlowControlLoopsCollections.cls

```diff
@@ -183,14 +247,15 @@ public with sharing class FlowControlLoopsCollections {
      */
     public static List<Integer> createAndPopulateList() {
         // Create a new list of integers
+        List<Integer> newList = new List<Integer>();
 
         // Add the integers 1, 2, and 3 to the list
-        // add 1
-        // add 2
-        // add 3
-
+        newList.add(1);// add 1
+        newList.add(2);// add 2
+        newList.add(3);// add 3
+        system.debug(newList);
         // Return the populated list
-        return null; // Replace null with the variable you used to store the result
+        return newList; // Replace null with the variable you used to store the result
```

Comment:
Good job. A senior dev tip here is that  if you know the values of your list at write time. You can add them in the List constructor. 

```
 public static List<Integer> createAndPopulateList() {
      return new List<Integer>{1, 2, 3};
    }
 ```